packages/lab/src/__tests__/__e2e__/form-field-next/FormFieldNext.cy.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  FormFieldNext,
+  FormFieldLabel,
+  FormFieldHelperText,
+  InputNext,
+} from "@salt-ds/lab";
+
+const MockChildren = () => {
+  return (
+    <>
+      <FormFieldLabel>Label</FormFieldLabel>
+      <div />
+      <FormFieldHelperText>Helper text</FormFieldHelperText>
+    </>
+  );
+};
+
+describe("GIVEN a FormFieldNext", () => {
+  describe("WHEN FormFieldLabel is provided", () => {
+    it("THEN the helperText is rendered", () => {
+      cy.mount(
+        <FormFieldNext>
+          <MockChildren />
+        </FormFieldNext>
+      );
+
+      cy.findByText("Label").should("exist");
+    });
+  });
+
+  describe("WHEN FormFieldHelperText is provided", () => {
+    it("THEN the helperText is rendered", () => {
+      cy.mount(
+        <FormFieldNext>
+          <MockChildren />
+        </FormFieldNext>
+      );
+
+      cy.findByText("Helper text").should("exist");
+    });
+  });
+
+
+  describe("WHEN an id is provided", () => {
+    it("THEN the label and helper text should have the corresponding ids", () => {
+      cy.mount(
+        <FormFieldNext id={"test-id"}>
+          <MockChildren />
+        </FormFieldNext>
+      );
+
+      cy.findByText("Label").should(
+        "have.attr",
+        "aria-labelledby",
+        "label-test-id"
```

Comment:
Interesting to see id used as suffix instead of prefix. This is also worth to mention in the dev documentation, around testing by consumer.